releng/setup-env.sh

```diff
@@ -6,6 +6,10 @@ build_platform=$(uname -s | tr '[A-Z]' '[a-z]' | sed 's,^darwin$,macos,')
 build_arch=$($releng_path/detect-arch.sh)
 build_platform_arch=${build_platform}-${build_arch}
 
+if [ -z "$PYTHON3" ]; then
+  PYTHON3 = `which python3`
+fi
+
```

Comment:
- Would be good to move this down to right before `if [ $host_platform = android ]; then`, to avoid chopping the build/host section in two.
- Should drop spaces around `=`.
- For consistency I'd say we should introduce a "local" variable, something like:

```sh
if [ -n "$PYTHON3" ]; then
  python3=$PYTHON3
else
  python3=$(which python3)
fi
```

Reply:
All changed, building on Mac to see how it goes. No luck with `+ > /dev/null 2>&1` to `\;` change. Sed gives a wall of text errors on Linux.